challenge.js

```diff
@@ -34,14 +37,16 @@
  *	Using the same HTML elements as above...
  *
  *	Suppose A is the body element, B, C, and D are all divs,
- *		and E is a p. 
+ *		and E is a p.
  *
  *	Write a CSS selector that can be used to retrieve E from the DOM.
  *
  *	Answer below.
  */
 
-// ANSWER
+// ANSWER - do not know this one
+
+  body div p {}
```

Comment:
`body div p` would select E. It would look for any p that is a descendant (not necessarily a child) of a div that is a descendant of the body.

`body > div > p` would select any p that is a child of a div that is a child of the body, which will be a narrower search. This question was asking for any selector that would retrieve E and not necessarily only E.

Note that the curly braces there are strictly for CSS styling. You can feed `"body div p"` to `document.querySelectorAll` to retrieve all elements that the following styles within the curly braces would have been applied to: `document.querySelectorAll("body div p")`. 1